liquibase-core/src/main/java/liquibase/integration/commandline/LiquibaseCommandLineConfiguration.java

```diff
@@ -63,12 +64,18 @@
         LOG_FILE = builder.define("logFile", File.class).build();
         OUTPUT_FILE = builder.define("outputFile", File.class).build();
 
+        MONITOR_PERFORMANCE = builder.define("monitorPerformance", String.class)
+                .setDescription("Enable performance tracking. Set to 'false' to disable. If set to 'true', data is stored to a `liquibase-TIMESTAMP.jfr` file in your working directory. Any other value will enable tracking and be used as the name of the file to write the data to.")
```

Comment:
we can include the jdk requirements here too.

Reply:
@nvoxland looks like @neupsh has some suggestions; could you review? Thanks!

Reply:
It should work with any Java 11+ version, and nothing under 11 is supported by oracle so I didn't think it was worth calling out in there. If someone tries to run with an old version it will tell them at the time which is probably good enough?